src/components/organisms/NavBarSchedule/NavBarSchedule.tsx

```diff
@@ -97,11 +101,15 @@ export const NavBarSchedule: FC<NavBarScheduleProps> = ({ isVisible }) => {
           className={classes}
           onClick={onWeekdayClick}
         >
-          {dayIndex === 0 ? "Today" : format(day, "E")}
+          {isScheduleStartDate
+            ? format(day, "E, LLL d")
+            : dayIndex === 0
+            ? "Today"
+            : format(day, "E")}
```

Comment:
In the main body of the code before this JSX (probably directly after `firstDayOfSchedule` is defined) I would do something like:

```typescript
const firstDayOfScheduleLabel = isToday(firstDayOfSchedule) ? "Today" : format(day, "E, LLL d");
```

Then change the code here to:

```suggestion
          {dayIndex === 0 ? firstDayOfScheduleLabel : format(day, "E")}
```

Further Reading:

- https://date-fns.org/v2.21.3/docs/isToday
- https://date-fns.org/v2.21.3/docs/format
  - > Day of week (formatting) | E..EEE | Mon, Tue, Wed, ..., Sun
  - > Month (stand-alone) | LLL | Jan, Feb, ..., Dec
  - > Day of month | d | 1, 2, ..., 31

Reply:
I had a bit of trouble with this one, and left (in a slightly modified form) as is. As I understand it, the proposed change would render every day other than the first as a day of the week. The aim here is to render all days as 'Mon, Sep 1' if the start date is selected, and 'Today', 'Mon', ... if the start date of the schedule is driven by the current day

Reply:
> The aim here is to render all days as 'Mon, Sep 1' if the start date is selected, and 'Today', 'Mon', ... if the start date of the schedule is driven by the current day

Mm, fair. Nesting ternaries like you did makes it super hard to read the code/infer the intended logic, which is why I made the logic error that lead to my suggestion.